docs/manage/task-config-policies.rst

```diff
@@ -0,0 +1,157 @@
+.. _task-config-policies:
+
+#######################
+ Config Policies Guide
+#######################
+
+Config Policies allow administrators to set limits on how users can define workloads (e.g.,
+experiments and notebooks, tensorboards, shells, and commands). This feature enables administrators
+to govern user behavior more closely at the workspace and cluster level. Administrators can include
+any parameter referenced in the :ref:`experiment configuration <experiment-config-reference>`.
+
+.. include:: ../_shared/attn-enterprise-edition.txt
+
+When implementing Config Policies, administrators should be aware of the following:
+
+#. Separate Priority Limits: Administrators can now set different priority limits for experiments
+   and NTSC (notebooks, tensorboards, shells, and commands) tasks. This allows for more granular
+   control over task prioritization and resource allocation.
+
+#. Priority Overrides: If a given scope has an invariant config policy set for a given task, and
+   that invariant configuration policy specifies ``priority``, this priority can still be overridden
+   using the ``SetJobPriority`` API endpoint. However, the new priority must not violate any
+   constraints defined in the applicable policies for that scope.
+
+These features provide administrators with greater flexibility in managing task priorities while
+still maintaining policy-based control over resource usage.
+
+**************
+ Key Features
+**************
+
+-  Set policies via WebUI or CLI
+-  Define limits for resource usage, environment settings, and more
+-  Apply policies at different levels (cluster-wide or workspace)
+-  Override capabilities for specific user roles or groups
+-  Set different priority limits for experiments and NTSC (notebooks, TensorBoards, shells, and
+   commands) tasks
+-  Allow priority overrides within policy constraints
+
+******************
+ Setting Policies
+******************
+
+Administrators can set Config Policies using either the WebUI or the CLI.
+
+WebUI
+=====
+
+#. Sign in to the Determined WebUI as a cluster administrator or workspace administrator.
+#. Navigate to **Config Policies**.
```

Comment:
clarify that there's a cluster-wide Config Policies page and a Config Policies tab for each workspace page?

Reply:
two navigation path?

Reply:
@johnkim-det please check if resolved